src/Storages/StorageMongoDB.cpp

```diff
@@ -39,7 +39,7 @@ StorageMongoDB::StorageMongoDB(
     const String & comment)
     : IStorage(table_id_)
     , host(host_)
-    , port(port_)
+    // , port(port_)
```

Comment:
Remove this
you can also remove host and options fields

Reply:
Alright, done.